### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Jellyfin.Api/Controllers/DynamicHlsController.cs [hunk 1]
```diff
@@ -1144,30 +1144,30 @@ private async Task<ActionResult> GetVariantPlaylistInternal(StreamingRequestDto
...
-            builder.AppendLine("#EXTM3U");
-            builder.AppendLine("#EXT-X-PLAYLIST-TYPE:VOD");
-            builder.AppendLine("#EXT-X-VERSION:3");
-            builder.AppendLine("#EXT-X-TARGETDURATION:" + Math.Ceiling(segmentLengths.Length > 0 ? segmentLengths.Max() : state.SegmentLength).ToString(CultureInfo.InvariantCulture));
-            builder.AppendLine("#EXT-X-MEDIA-SEQUENCE:0");
+            builder.AppendLine("#EXTM3U")
+                .AppendLine("#EXT-X-PLAYLIST-TYPE:VOD")
+                .AppendLine("#EXT-X-VERSION:3")
+                .Append("#EXT-X-TARGETDURATION:")
+                .Append(Math.Ceiling(segmentLengths.Length > 0 ? segmentLengths.Max() : state.SegmentLength))
```
Comment (cvium) [inline Jellyfin.Api/Controllers/DynamicHlsController.cs:1151]: Shouldn't this just be AppendLine?

File: Jellyfin.Api/Controllers/DynamicHlsController.cs [hunk 2]
```diff
@@ -1144,30 +1144,30 @@ private async Task<ActionResult> GetVariantPlaylistInternal(StreamingRequestDto
...
-                        "hls1/{0}/{1}{2}{3}",
-                        name,
-                        index.ToString(CultureInfo.InvariantCulture),
-                        segmentExtension,
-                        queryString));
-
-                index++;
+                builder.Append("#EXTINF:")
+                    .Append(length.ToString("0.0000", CultureInfo.InvariantCulture))
+                    .Append(", nodesc")
```
Comment (cvium) [inline Jellyfin.Api/Controllers/DynamicHlsController.cs:1163]: Should be AppendLine?

File: Jellyfin.Api/Controllers/DynamicHlsController.cs (hunk 1 above)
Comment (Bond-009) [reply Jellyfin.Api/Controllers/DynamicHlsController.cs:1151]: `AppendLine` only works with strings

File: Jellyfin.Api/Controllers/DynamicHlsController.cs (hunk 2 above)
Comment (Bond-009) [reply Jellyfin.Api/Controllers/DynamicHlsController.cs:1163]: yep

**Review by crobibero — APPROVED** [review]

**Review by cvium — APPROVED** [review]

**Pull request merged**